src/content/docs/en/community-resources/content.mdx

```diff
@@ -37,11 +37,9 @@ Check out the following courses and tutorials to learn more about Astro.
 
 ## Recipes and Guides
 
-See guided examples of adding features to your Astro project, submitted by our community members!
+[Astro's official recipes](/en/recipes/) are short, focused how-to guides that walk a reader through completing a working example of a specific task. Recipes are a great way to add new features or behavior to your Astro project by following step-by-step instructions!
```

Comment:
```suggestion
[Astro's official recipes](/en/recipes/) are quick, focused how-to guides that walk a reader through completing a working example of a specific task. Recipes are a great way to add new features or behavior to your Astro project by following step-by-step instructions!
```
I think this would make it sound like the recipes are concise, rather than simple, and perhaps encourage more people to use them. I might ALSO be overthinking!

Reply:
Our official recipes ARE concise! Quick refers to speed, and there's no guarantee how long it takes you to do them. But the ones we have are actually pretty short!